src/applications/gi/tests/e2e/gibct-helpers.js

```diff
@@ -117,7 +116,9 @@ const selectSearchResult = (client, facilityCode, checkLocation = true) => {
   if (checkLocation) expectLocation(client, `/profile/${facilityCode}`);
 };
 
-const createAccordionId = name => `#${UtilHelpers.createId(name)}-accordion`;
+const createId = name => name?.toLowerCase().replace(/\s/g, '-');
```

Comment:
Once `import environment from 'platform/utilities/environment';` is removed from `src/applications/gi/utils/helpers.js` changes can be reverted